src/snowflake/cli/plugins/nativeapp/exceptions.py

```diff
@@ -5,11 +5,11 @@
 
 
 class ApplicationPackageAlreadyExistsError(ClickException):
-    """An application package not created by SnowCLI exists with the same name."""
+    """An application package not created by Snowflake CLI exists with the same name."""
```

Comment:
Is this the "official" name? I'll make sure to use that in the NA docs as well.

Reply:
Yes, this is the official name.